### ISSUE TITLE
pab1

### ISSUE THREAD

**Issue by Antonialock** [pombase/curation#2357]
I don't think this is the way to do it (it doesn't make sense when your read it out - wee1 is a kinase).

<img width="945" alt="Screenshot 2019-04-04 at 21 11 10" src="https://user-images.githubusercontent.com/12727506/55585292-3d43ce00-571e-11e9-9242-02d80929fdb8.png">

I would do

pab1
-protein phosphatase activator activity
has_substrate ppa2 part_of regulation of G2/M (or whatever)

ppa2
-protein phosphatase activity
has_substrate wee1/cdc25 part_of regulation of G2/M (or whatever)

the fact that pab1 lends specificity to wee1/cdc25 will be evident from the involved_in process

and possibly
pab1
-protein phosphatase activity
qualifier contributes_to
has_substrate wee1/cdc25 part_of regulation of G2/M (or whatever)

**Comment by Antonialock** [comment]
updated to fix term copy paste  typo

**Comment by ValWood** [comment]
yup, that's the gene product name not the specific MF... on my pile...

**Comment by ValWood** [comment]
Yes I added this note to this session

well, this doesn't really show that this is "directly" responsible. But the consensus appears to be that it is, because it is conserved in other species. I was waiting to plug this hole to get the full pathway for this (as depicted in Sergio's cell paper) see https://github.com/pombase/curation/issues/1225 so I kept it, as a MF, has substrate, added contributes to because everyone seems to look at this PPA2-cdc55 ortholog regulatory subunit rather than the phosphatase...


but I think the way you suggest is a better way to do this.

I have a number of tickets open with a similar issue, when the substrate was isentified using a regulator ratehr than the modifying subunit.
